### ISSUE TITLE
Clarify where to specify data source addresses for non-Portland Metro builds

### ISSUE THREAD

**Issue by williamflynt** [pelias/kubernetes#114]
Hey team! 

I was trying to understand how to build a USA-only instance of Pelias and I noticed there is a lack of "direction" on how to specify data sources.

---
#### Here's what I was expecting :sparkles:

A documented set of steps and values to downloading data and specifying the resource addresses.

---
#### Here's what I think could be improved :trophy:

I can think of a few solutions to this problem. An experienced contributor's opinion on best practices for the Pelias ecosystem would be appreciated.

* Clear documentation on what to change for your specific build, using the existing templates.
* A separate object in the `values.yaml` files containing clearly defined `dataSources`.
* A separate `data-sources.yaml` that's imported to the templates.

It would really help to get input on this, as I'm still not clear on what needs changed to specify custom data repositories. I'm happy to implement and submit the PR!

**Comment by missinglink** [comment]
Yes I totally agree, I think as project maintainers we often take knowledge like this for granted.

**Comment by missinglink** [comment]
In fact the whole `pelias.json` is not very well documented, or at least the documentation is spread over several repos.

I'll give a quick summary of the import sources:

**geonames**
the country code defines which one of their files is downloaded, omitting a country code downloads the file which includes everything.
https://download.geonames.org/export/dump/

**openaddresses**

These filenames are from the website http://results.openaddresses.io/?runs=all#runs on the left, IIRC they must be specified with the .CSV extension although the downloader is smart enough to know to download a .ZIP file of the same name.

**openstreetmap**

This is simply any .OSM.PBF file, usually from an extract provider like Geofabrik or from the OSM planet website.

**whosonfirst**

There is a property which allows us to filter their dataset for a region, you need to give it a WOF ID and the code takes care of enumerating all the records above and below that ID in the hierarchy.
These ID values can be found from an existing Pelias instance (such as geocode.earth) or via https://spelunker.whosonfirst.org/ or the placeholder demo link found on the pelias/placeholder repo.

**TIGER**

Is only applicable to the USA and only when building the interpolation service, there is more information in the readme about how to select states/counties.

I think in general all of this is documented in the individual importer repo readme but it's fairly fragmented right now making it hard to follow.

**Comment by missinglink** [comment]
[edit] got my wires crossed as to which repo this issue was on, so I've deleted my commented and moved them to https://github.com/pelias/docker/issues/142

**Comment by KristinaHus** [comment]
Hi, how can I add a pelias.json file to Kubernetes configurations?
